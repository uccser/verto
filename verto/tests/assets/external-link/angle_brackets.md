Check out this [website](<https://en.wikipedia.org/wiki/Entropy_(information_theory)>).
