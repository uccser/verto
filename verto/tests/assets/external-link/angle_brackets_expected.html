<p>Check out this <a href="https://en.wikipedia.org/wiki/Entropy_(information_theory)" target="_blank">website</a>.</p>
